presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -44,6 +44,7 @@
     private Duration checkpointInterval = new Duration(60000, TimeUnit.MILLISECONDS);
     private String logicalProcessName = "process1";
     private int iteratorNumber;
+    private Duration updateInterval = new Duration(600000, TimeUnit.MILLISECONDS);
```

Comment:
nit: Move to L30 and simplify:
```suggestion
    private Duration updateInterval = new Duration(1, MINUTES);
```